hail/python/test/hail/conftest.py

```diff
@@ -0,0 +1,21 @@
+import hashlib
+import os
+
+import pytest
+
+
+def pytest_collection_modifyitems(config, items):
+    n_splits = int(os.environ.get('PYTEST_SPLITS', '1'))
+    split_index = int(os.environ.get('PYTEST_SPLIT_INDEX', '-1'))
+    if n_splits <= 1:
+        return
+    if not (0 <= split_index < n_splits):
+        raise RuntimeError(f"invalid split_index: index={split_index}, n_splits={n_splits}\n  env={os.environ}")
+    skip_this = pytest.mark.skip(reason="skipped in this round")
+
+    def digest(s):
+        return int.from_bytes(hashlib.md5(str(s).encode('utf-8')).digest(), 'little')
+
+    for item in items:
+        if not digest(item.name) % n_splits == split_index:
```

Comment:
Any reason why `hash(item.name)` is insufficient?

Reply:
Python doesn't use a fixed hash seed for security reasons. We could require a static PYTHONHASHSEED as a precondition for using this stuff, but seems a bit more brittle